### DISCUSSION TITLE
Does this plugin support a slider or range from bigger number to smaller number

### DISCUSSION THREAD

**Discussion by afwansyawal** [IonDen/ion.rangeSlider#757] [Q&A] [answered]
Does this plugin support a slider or range from bigger number to smaller number? 

In some scenarios, smaller number can means as a higher result. For example, a ranking system. Suppose that I have a range 10 to 1. I tried switching the min max value, 10 in the min and 1 in the max. It does not working.

How can I achieve this using this plugin? 
Thank you

**Comment by IonDen** [comment, marked as answer]
Where are multiple ways to do it.

Using prettify function:
https://jsfiddle.net/IonDen/q5phxkb3/

Using values array:
https://jsfiddle.net/IonDen/e7v5mnx3/

**Reply by afwansyawal** [reply]
Wow, Thank you very much for your help
